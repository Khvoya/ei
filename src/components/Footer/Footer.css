.Footer {
    display: flex;
    flex-direction: row;
    padding: 10px 0;
    align-content: center;
    justify-content: center;
}
.Footer__bg {
  background-color: black;
  box-shadow: 0 0 6px 5px white;
  color: white;
}
.Footer__wrap {
  width: 100%;
}
.Footer__social-wrap {
  display: flex;
  justify-content: space-between;
}
.Footer__social-icon {
  width: 30px;
  height: 30px;
  margin: 5px;
}

.Footer__social,
.Footer__coop,
.Footer__year {
  margin: 10px 0;
  display: flex;
  justify-content: space-around;
  align-items: center;
}
@media (min-width: 768px) and (max-width: 991px) {
  .Footer {
    font-size: 13px;
  }
}
