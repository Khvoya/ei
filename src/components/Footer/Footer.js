import React from "react";
import {Container, Col, Row} from 'react-bootstrap';
import {socialData} from "constants/headerData";
import "./Footer.css";

const Footer = () => {
    const socialIcons = socialData.map(icon => (
      <a href={icon.href} target="blank" key={icon.name}>
        <img className="Footer__social-icon" src={icon.img} alt={icon.name}/>
      </a>
    ));
    return (
      <div className="Footer__bg">
        <Container className="Footer">
          <Row className="Footer__wrap">
            <Col className="Footer__coop" lg={4} xl={4} md={4} >
              <a href="mailto:electricindiansband@gmail.com">electricindiansband@gmail.com</a>
            </Col>
            <Col className="Footer__social" lg={4} xl={4} md={4} >
              <div className="Footer__social-wrap">
                {socialIcons}
              </div>
            </Col>
            <Col className="Footer__year" lg={4} xl={4} md={4} sm={12} >
              <span className="Footer__dev-info-r">Electric Indians 2019 Â©</span>
            </Col>
          </Row>
        </Container>
      </div>
    );
};

export default Footer;
