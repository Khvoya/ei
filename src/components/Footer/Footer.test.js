import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer', () => {

  it('should render correctly', () => {
    const output = shallow(
      <Footer />
    );
    expect(output).toMatchSnapshot();
  });
});
