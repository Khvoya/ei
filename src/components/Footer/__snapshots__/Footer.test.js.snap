// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<div
  className="Footer__bg"
>
  <Container
    as="div"
    className="Footer"
    fluid={false}
  >
    <Bootstrap(Row)
      className="Footer__wrap"
    >
      <Col
        as="div"
        className="Footer__coop"
        lg={4}
        md={4}
        xl={4}
      >
        <a
          href="mailto:electricindiansband@gmail.com"
        >
          electricindiansband@gmail.com
        </a>
      </Col>
      <Col
        as="div"
        className="Footer__social"
        lg={4}
        md={4}
        xl={4}
      >
        <div
          className="Footer__social-wrap"
        >
          <a
            href="https://www.instagram.com/electricindians/"
            key="Instagram"
            target="blank"
          >
            <img
              alt="Instagram"
              className="Footer__social-icon"
              src="instagram-logo.svg"
            />
          </a>
          <a
            href="https://www.facebook.com/electricindians/"
            key="Facebook"
            target="blank"
          >
            <img
              alt="Facebook"
              className="Footer__social-icon"
              src="facebook-logo-button.svg"
            />
          </a>
          <a
            href="https://vk.com/electricindians"
            key="Vk"
            target="blank"
          >
            <img
              alt="Vk"
              className="Footer__social-icon"
              src="vk-social-logotype.svg"
            />
          </a>
          <a
            href="https://www.youtube.com/channel/UCab_uVuxFhrwyARbotaz4Tg"
            key="Youtube"
            target="blank"
          >
            <img
              alt="Youtube"
              className="Footer__social-icon"
              src="youtube-logotype.svg"
            />
          </a>
          <a
            href="https://electricindians.bandcamp.com/"
            key="Bandcamp"
            target="blank"
          >
            <img
              alt="Bandcamp"
              className="Footer__social-icon"
              src="bandcamp-logo.svg"
            />
          </a>
          <a
            href="https://twitter.com/Electricindian2"
            key="Twitter"
            target="blank"
          >
            <img
              alt="Twitter"
              className="Footer__social-icon"
              src="twitter-logo-button.svg"
            />
          </a>
        </div>
      </Col>
      <Col
        as="div"
        className="Footer__year"
        lg={4}
        md={4}
        sm={12}
        xl={4}
      >
        <span
          className="Footer__dev-info-r"
        >
          Electric Indians 2019 Â©
        </span>
      </Col>
    </Bootstrap(Row)>
  </Container>
</div>
`;
